dpdata/unit.py

```diff
@@ -0,0 +1,104 @@
+from abc import ABC
+
+AVOGADRO = 6.02214179E23    # Avagadro constant
+ELE_CHG  = 1.602176487E-19  # Elementary Charge, in C
```

Comment:
We may use `scipy.constants.Avogadro` and `scipy.constants.elementary_charge`.

Reply:
What does this constant come from? As is known, some physical constants were measurements before 2019 so they are updated every four years. For example, `ELE_CHG` in 2014 is 1.602 176 6208(98) × 10−19. See [here](https://physics.nist.gov/cuu/pdf/nonsi_2014.pdf). But after the definition of some units were [changed in 2019](https://doi.org/10.1038/d41586-019-01614-8), now `ELE_CHG` is [an exact value](https://physics.nist.gov/cgi-bin/cuu/Value?evj) (1.602 176 634  x 10-19 J). I think what you use here is even older than 2014?

Reply:
`scipy.constants.elementary_charge` uses the latest defination.